automation_tools/__init__.py

```diff
@@ -2320,7 +2278,7 @@ def product_install(distribution, create_vm=False, certificate_url=None,
         execute(setup_bfa_prevention, host=host)
     execute(fix_qdrouterd_listen_to_ipv6, host=host)
 
-    if create_vm and 'base' in target_image:
+    if os.environ.get('PYTHON_CODE_COVERAGE') == 'true':
```

Comment:
can we avoid setting it as env var? pass as args ?

Reply:
we can, I did it the same way as is `RUBY_CODE_COVERAGE` env var

was thinking about just one CODE_COVERAGE but Ruby code coverage has some issues that's why it is turned off, so couldn't merge python + ruby flag easily...